packages/documentation-framework/templates/mdx.js

```diff
@@ -84,6 +85,11 @@ const MDXChildTemplate = ({
           {' '}To learn more about the process, visit our <Link to="/get-started/about#major-release-cadence">about page</Link>.
         </InlineAlert>
       )}
+       {(template || source === 'react-template') && (
+        <InlineAlert title="Templates" variant="info">
+          {`This page showcases templates for the ${id.toLowerCase()} component. A template consists of the component and additional logic supporting the use case, and has a streamlined API with some limited customization.`}
```

Comment:
```suggestion
          {`This page showcases templates for the ${id.toLowerCase()} component. A template combines a component with logic that supports a specific use case, with a streamlined API that offers additional, limited customization.`}
```
@kmcfaul thanks for clarifying -- I'm following better now! wdyt about this phrasing?

Reply:
Sounds great!